skimage/__init__.py

```diff
@@ -68,10 +68,35 @@
 
 """
 
-import sys
+__version__ = '0.19.0.dev0'
 
+__all__ = [
+    'color',
+    'data',
+    'draw',
+    'exposure',
+    'external',
+    'feature',
+    'filters',
+    'future',
+    'graph',
+    'io',
+    'measure',
+    'metrics',
+    'morphology',
+    'registration',
+    'restoration',
+    'segmentation',
+    '_shared',
+    'transform',
+    'util',
+    'viewer'
+]
+
+from .util.lazy import install_lazy
+__getattr__, __dir__, _ = install_lazy(__name__, __all__)
```

Comment:
Does this work with Python 3.5?

Reply:
No, but we only support 3.7+?

Reply:
I presume it works but it doesn't do anything.  But Python 3.5 is ancient news now.

Reply:
I'm asking this in the context that on line 77 /102 we have the lines `ensure_python_version((3, 5))

We tried to ensure that we had a nice message to the end users, instead of a scary python stack trace.

![image](https://user-images.githubusercontent.com/90008/119204135-aacc2900-ba62-11eb-8337-01197e4abce5.png)

Reply:
Yeah, it seems the `ensure_python_version` call below should either be removed or updated to use `(3, 7)` instead of `(3, 5)`. If updated, it may need to be called prior to `install_lazy`?

Reply:
Ah, thanks, I get what you're saying now @hmaarrfk !